src/main/resources/standardTestCases/jes_gcs_root/jes_gcs_root.options

```diff
@@ -0,0 +1,3 @@
+{
+  "jes_gcs_root": "gs://cloud-cromwell-dev/centaur-tests/jes_gcs_root"
```

Comment:
Tests depending on our buckets shouldn't be in the centaur repo. We can run them via the arbitrary test mechanism.